dist/sampling.js

```diff
@@ -23,7 +23,7 @@
   var rate = 10;
   var daysValid = 90;
   if (sample(rate, daysValid)) {
-    console.log('REPLACE THIS LINE WITH YOUR FULLSTORY SNIPPET');
+    console.log('REPLACE THIS ENTIRE LINE AND CONSOLE LOG WITH YOUR FULLSTORY SNIPPET');
```

Comment:
@jakeh2os Can you also change the `src/sampling.js` file.  It's the exact same line of code, but when you run `npm run build` the `dist/sampling.js` file gets created.  So best practice is to edit the `src` files and then build to produce the `dist` files.